### ISSUE TITLE
Some labels review

### ISSUE THREAD

**Issue by tdipisa** [geosolutions-it/ckanext-dcatapit#158]
Can we change the Tag label with 'Keyword' ('Parola chiave' in italian)?

**Comment by etj** [comment]
We have to check if CKAN allows the default translations to be overriden.

**Comment by cezio** [comment]
Where exactly in UI this should be changed?

**Comment by tdipisa** [comment]
@cezio, 

![tag1.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5943f6b8abdc49001a6611c9/125516c66e82c728ace21e0d46fc959d72c0cfeb9ffbb049a15ee4d67304722a2b05fa2874d0b72ab91579575f5f4ca9120b5f43faf89b37b4f46d2ede0356b49b381d66cc2f10a5b2831abd01bf21834d85b4b48248d2a5.png)

![tag2.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5943f6b8abdc49001a6611c9/125516c66e82c728ace21e0d46fc959d71c0cfeb9ffbb049a15ee4d67304722a2b05fa2874d0b72ab91579575f5f4ca9120b5f43faf89b37b4f46d2ede0356b49b381d66cc2f10a4b2831abd01bf21834d85b0b98d4ed2a5.png)

**Comment by cezio** [comment]
That would be a bit tricky. I tried to use the same translation domain as ckan, but it didn't work. I could replace this with js, after whole markup is loaded, but that would need some custom translation infrastructure (per-language value mapping).

**Comment by tdipisa** [comment]
@cezio probably we need to extend the involved templates and put there the custom labels using the dcatapit i18n files. What do you think ?

**Comment by cezio** [comment]
I'm not sure if those values are  comming from templates directly (at least not all). Facet names are from https://github.com/ckan/ckan/blob/ckan-2.5.7/ckan/controllers/package.py#L238 probably. Localized tags are coming from ckanext-mulitlang apparently.

**Comment by tdipisa** [comment]
@cezio so you mean that is not possible to simply override for example the msgid [here](https://github.com/ckan/ckan/blob/master/ckan/i18n/it/LC_MESSAGES/ckan.po#L920).
The 'Tag' word occurs several times into the ckan.po file so if we cannot do that symply overriding the base .po entries leaving it as it is is better. We can not introduce heavy customizations for this.

**Comment by giorgialodi** [comment]
@tdipisa I agree with you. If it is not possible to follow what you are suggesting  but we need to heavly customize it, leave it as it is with "tag" label

**Comment by cezio** [comment]
There are several obstacles. Extensions are using different translation domains, so by default translations are isolated. The other thing is, even if extension will use `ckan` domain (used by ckan itself) instead of it's own, gettext will use first .mo file it will find. and this is usually ckan's own. 

However, maybe there's some way to hack i18n infrastructure and add our catalog before ckan's. The problem here is that ckan 2.5 uses pylons.i18n, while latest uses flask-babel (but both use gettext underneath).

**Comment by tdipisa** [comment]
@cezio I'm going to block this for the moment.
